packages/console/src/components/AppContent/components/Sidebar/components/Contact/index.tsx

```diff
@@ -31,8 +31,8 @@ const Contact = ({ isOpen, onCancel }: Props) => {
                 <ContactIcon />
               </div>
               <div className={styles.text}>
-                <div className={styles.title}>{t(title)}</div>
-                <div className={styles.description}>{t(description)}</div>
+                <div className={styles.title}>{String(t(title))}</div>
```

Comment:
Why is this conversion required?

Reply:
Seems this is related and `react-i18next v11.16.4` fixed it
https://github.com/i18next/react-i18next/issues/1476

Reply:
<img width="1320" alt="image" src="https://user-images.githubusercontent.com/14722250/183123361-69573c46-4900-4575-9182-a7df10cc7425.png">

upgraded to the latest version it still doesn't work though. can wait for the fix

Reply:
i think it's fine - since `t` can return an object - which is not a valid React node

Reply:
Yeah, I also tried the latest version last night, as well as the d.ts file they suggested, it didn't work... SMH